playwright/_impl/_transport.py

```diff
@@ -97,6 +98,11 @@ async def wait_until_stopped(self) -> None:
 
     async def run(self) -> None:
         self._stopped_future: asyncio.Future = asyncio.Future()
+        platform = sys.platform
+        if platform == "win32" and sys.stdout is None:
+            creationflags = subprocess.CREATE_NO_WINDOW
+        else:
+            creationflags = 0
```

Comment:
```suggestion
        creationflags = 0
        if sys.platform == "win32" and sys.stdout is None:
            creationflags = subprocess.CREATE_NO_WINDOW
```
nit: style preference

Reply:
Sorry I'm not sure how I did it, I wanted to merge with your suggestion, and now the add suggestion to batch button is unavailable. I'm new to github.

Reply:
No worries, I applied it. The reason was probably because you resolve it already, thats why i can't get applied again.